### ISSUE TITLE
Create grid offset

### ISSUE THREAD

**Issue by marciovicente** [jusbrasil/monalisa#39]
Currently I'm using a fake grid offset, like:
`<div class="grid-col-6">&bnsp;</div> <div class="grid-col-6">My content</div>`

Let's try implement a generic offset to each column. Eg: `<div class="grid-col-6 grid-offset-6">My content</div>`

**Comment by rafaelverger** [comment]
something to emulate `colspan`?
```html
<tr>
  <td>One</td><td>Two</td>
</tr>
<tr>
  <td colspan=2>Three</td>
</tr>
```
![image](https://cloud.githubusercontent.com/assets/588697/6971494/380a41b4-d953-11e4-9ffb-1ed2325e5100.png)

**Comment by shankarcabus** [comment]
No, @rafaelverger... Something to create a "new fake column". Hmm, like a empty column after/before a column. Did you know?
